### ISSUE TITLE
ModuleNotFoundError: No module named '_lzma'

### ISSUE THREAD

**Issue by liyanhui1228** [google/pybadges#3]
Got `ModuleNotFoundError` when using this package:

```
Python 3.6.3 (default, Feb 16 2018, 11:33:20) 
[GCC 6.3.0 20170516] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from pybadges import badge
>>> s = badge(left_text='coverage', right_text='23%', right_color='red')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/google/home/yanhuil/fork/cloud-opensource-python/.tox/py36/lib/python3.6/site-packages/pybadges/__init__.py", line 116, in badge
    precalculated_text_measurer.PrecalculatedTextMeasurer
  File "/usr/local/google/home/yanhuil/fork/cloud-opensource-python/.tox/py36/lib/python3.6/site-packages/pybadges/precalculated_text_measurer.py", line 79, in default
    import lzma
  File "/usr/local/lib/python3.6/lzma.py", line 27, in <module>
    from _lzma import *
ModuleNotFoundError: No module named '_lzma'
```

**Comment by CSHS** [comment]
The XZ Utils package is missing.  Installing it, then installing Python3.x should resolve this issue.  Hope this helps!

Sources: 
-https://blastedbio.blogspot.com/2012/09/installing-python-33-with-lzmaxy-on-mac.html (says Mac, but general flow of instructions should apply to Linux as well)
-http://tukaani.org

**Comment by brianquinlan** [comment]
Hey @liyanhui1228, does @CSHS's suggestion resolve the issue for you?

**Comment by brianquinlan** [comment]
_lzma should be included with Python

**Comment by brianquinlan** [comment]
It looks like Debian does not include a Python 3 version of python-lzma. It might be more practical to use a different compression method for now.

**Comment by brianquinlan** [comment]
Fixed in 1.02
